src/components/User/Tags/TagsManager.vue

```diff
@@ -0,0 +1,176 @@
+<template>
+  <div class="tags-manager">
+    <div class="tags-manager__form">
+      <input-field
+        v-model="tagName"
+        :placeholder="'tags-manager.add-tag-input-placeholder' | globalize"
+        :disabled="isPending"
+        class="tags-manager__input"
+        @blur="touchField('tagName')"
+        :error-message="getFieldErrorMessage('tagName', {
+          maxLength: TAG_NAME_MAX_LENGTH
+        })"
+      />
+      <button
+        class="app__btn tags-manager__btn"
+        @click="addTag(tagName)"
+      >
+        {{ "tags-manager.btn-add-tag" | globalize }}
+      </button>
+    </div>
+    <div class="tags-manager__list-wrp">
+      <template v-if="list && list.length">
```

Comment:
list.length and use structure for handling loading and failed request

Reply:
fixed